algorithms/src/Kokkos_Sort.hpp

```diff
@@ -596,10 +578,21 @@ std::enable_if_t<Kokkos::is_execution_space<ExecutionSpace>::value> sort(
   bin_sort.sort(exec, view);
 }
 
+#if defined(KOKKOS_ENABLE_SERIAL)
+template <class DataType, class... Properties>
+void sort(const Serial& space,
+          const Kokkos::View<DataType, Properties...>& view) {
+  space.fence("Kokkos::sort: Fence before sorting on the host");
+  auto first = Experimental::begin(view);
+  auto last  = Experimental::end(view);
+  std::sort(first, last);
```

Comment:
Should I use `parallel_policy` or `parallel_unsequenced_policy`?

Reply:
No